src/index.html

```diff
@@ -1,16 +1,39 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <label class="bigBar">
+      <form 
+        action="server.name" 
+        method="get"
+      >
+        <input 
+          type="search"
+          placeholder="Try &#34Los Angeles&quot"
+          data-qa="big"
+          class="bigSearch"
+        >
+      </form>
+    </label>
```

Comment:
I am changing my code according to your comments 1 form 2 inputs. But before i did 2 forms because of the task: Each search bar be inside its own `form` element. (from Read me file). I guess it's because for positioning 2 elements i am using margin top/bottom. And because of <label> tag. Margin part is also active while i am pointing on it. So now 1 element have active zone of it's own height + margin and another one is just

Reply:
no, there still should be two forms, you'd just mixed up with the order of tags
it is just an example, but the order is
```
<form>
  <label>
    <input/>
  </label>
  <label>
    <input/>
  </label>
  <label>
    <input/>
  </label>
</form>

<form>
  <label>
    <input/>
  </label>
</form>
```